openapi/SwarmCommon.yaml

```diff
@@ -89,6 +89,20 @@ components:
               connectedPeers:
                 type: object
 
+    ChequebookBalance:
+      type: object
+      properties:
+        totalBalance:
+          type: integer
+        availableBalance:
+          type: integer
+
+    ChequebookAddress:
+      type: object
+      properties:
+        address:
+          type: string
```

Comment:
this could be a custom eth address type. I added one in cashout PR, if that one merges first we can change it here. otherwise i'll change it in that PR.

Reply:
you can change it to `$ref: #/components/schemas/EthereumAddress` now.